### ISSUE TITLE
SYNTAX_ERROR Line Number:193

### ISSUE THREAD

**Issue by Paranoid19-Art** [Kava4/PUBG-LOGITECH-NO-RECOIL-SEASON-31#39]
"SYNTAX_ERROR Line Number: 193. I’m getting this error. Can you help me? I didn’t have this error a season ago."

**Comment by Kava4** [comment]
Can you try the new code ?

**Comment by Ikhari** [comment]
There is no error but for some reason, the recoil only goes down, not straight.

![image](https://github.com/user-attachments/assets/f0d5fe95-f04c-4c76-affe-c127823c43e8)

**Comment by dbromgone** [comment]
@Kava4 

I'm experiencing an issue with the script where, despite following the instructions, the in-game vertical sensitivity behaves incorrectly. Specifically:

 * I set the in-game Vertical Sensitivity Multiplier to 1 as directed.
 * I also set local SensSetting = 1 in the script, as per the instructions.

--##############  For Vertical Sensitivity Multiplier 1 Set the SensSetting to 1       #################--

However, when I activate the macro, the aim moves straight down very quickly. After some trial and error, I found that adjusting the in-game Vertical Sensitivity Multiplier to around 0.32 reduces the issue, but the aim still doesn't function as it should. While it no longer jumps down erratically, the aiming remains inaccurate and feels off.

Could this be a bug, or is there something I'm missing in the setup? Any help would be greatly appreciated!

**Comment by Paranoid19-Art** [comment]
> Can you try the new code ?

Yes, I tried it, but the code that worked in 28 seasons before with the G-203 mouse is currently not working even though I have activated it. There is no activation happening. Is there a different issue?

**Comment by Paranoid19-Art** [comment]
"The code is currently running. I solved it with the Logitech G Hub version."

**Comment by Ikhari** [comment]
> > Can you try the new code ?
> 
> Yes, I tried it, but the code that worked in 28 seasons before with the G-203 mouse is currently not working even though I have activated it. There is no activation happening. Is there a different issue?

try to restart your pc, and run as admin your G-Hub

**Comment by Ikhari** [comment]
i already try some test the aim just moves straight down very quickly, even change sensitivity still weird. 
@Kava4

**Comment by nandaldian** [comment]
> @Kava4
> 
> I'm experiencing an issue with the script where, despite following the instructions, the in-game vertical sensitivity behaves incorrectly. Specifically:
> 
>     * I set the in-game Vertical Sensitivity Multiplier to 1 as directed.
> 
>     * I also set local SensSetting = 1 in the script, as per the instructions.
> 
> 
> --############## For Vertical Sensitivity Multiplier 1 Set the SensSetting to 1 #################--
> 
> However, when I activate the macro, the aim moves straight down very quickly. After some trial and error, I found that adjusting the in-game Vertical Sensitivity Multiplier to around 0.32 reduces the issue, but the aim still doesn't function as it should. While it no longer jumps down erratically, the aiming remains inaccurate and feels off.
> 
> Is there anything else I should check ? Like FOV or Resolution?
> 
> Could this be a bug, or is there something I'm missing in the setup? Any help would be greatly appreciated!

@Kava4 im experiencing the same issue as this user. Im using logitech g pro mouse.
recoil only goes down very quick, dont feel like no recoil, i need to push my mouse up to counter it.
Same settings as the image in the ZIP, except SensSettings, used 1 as my vertical recoil is 1 in-game.

Any solution for this? or are we doing something bad?

thanks in advance!!!

**Comment by Kava4** [comment]
The problem is with the PUBG's randomized recoil. I'm trying to find a solution but I don't have time for testing.
